### DISCUSSION TITLE
Adjust the wave spacing between top and bottom.

### DISCUSSION THREAD

**Discussion by jason30704** [bbc/peaks.js#549] [Q&A]
Hello,
Is there a way to adjust the height of the upper and lower sides of the wave?
Like the red frames in the picture below.
Thanks.

![image](https://github.com/user-attachments/assets/23c9bb7d-3681-4739-815a-c9676d956c64)

**Comment by chrisn** [comment]
Does [`view.setAmplitudeScale()`](https://github.com/bbc/peaks.js/blob/master/doc/API.md#viewsetamplitudescalescale) do what you need?

**Reply by jason30704** [reply]
I know that function, but it's not what I need.
I want to be able to add a fixed size spacing above or below the waveform. (Similar to padding? )
But setAmplitudeScale() will have different results for waveforms with different Scales.
Some may be able to achieve large spacing, but others will still fill the entire View (no spacing).
